cli/tests/unit/console_test.ts

```diff
@@ -313,27 +313,27 @@ unitTest(function consoleTestStringifyCircular() {
     "JSON {}",
   );
   assertEquals(
-    stringify(console),
+    stringify(new Console(() => {})),
```

Comment:
Test case is faulty on main, run with `deno test --quiet and it fails because it is not bound`.
Doesn't matter in this case what the fixture is, stringify is what is being tested.

Reply:
Yes I added it with `wrapConsole` API. Good catch and thanks for fixing this.